source/patterns/@aws-solutions-constructs/core/lib/eventbridge-helper.ts

```diff
@@ -47,5 +47,6 @@ export function buildEventBus(scope: Construct, props: BuildEventBusProps): even
     const _eventBusName = props.eventBusProps.eventBusName || 'CustomEventBus';
     return new events.EventBus(scope, _eventBusName, props.eventBusProps);
   }
-  return;
+  // ESLint requires this return statement, so disabling SonarQube warning
+  return; // NOSONAR
```

Comment:
Fun! the tooling yelling at each other

Reply:
Fun is one of the adjectives that could be applied...